File: 1672-richest-customer-wealth/1672-richest-customer-wealth.java
class Solution {
    public int maximumWealth(int[][] accounts) {
        int max = 0, tmp;
        
        for(int i = 0; i < accounts.length; i++){
            tmp = 0;
            for(int j = 0; j < accounts[i].length; j++)
                tmp += accounts[i][j];
            if(tmp > max) 
                max = tmp; 
        }
        return max;
    }
}

File: 1672-richest-customer-wealth/1672-richest-customer-wealth.swift
class Solution {
    func maximumWealth(_ accounts: [[Int]]) -> Int {
        var maxWealth = 0
        
        for account in accounts {
            var sum = account.reduce(0, {return $0 + $1})
            if sum > maxWealth {
                maxWealth = sum
            }
        }
        
        return maxWealth
    }
}